src/main/java/core/basesyntax/fileoperations/ReportGeneratorImpl.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax.fileoperations;
+
+import java.util.Map;
+
+public class ReportGeneratorImpl implements ReportGenerator {
+    @Override
+    public String getReport(Map<String, Integer> storage) {
+        StringBuilder report = new StringBuilder("fruit,quantity\n");
```

Comment:
we almost always wrap literals to constants, create REPORT_HEADER
use system.lineSeparator instead of /n everywhere